### ISSUE TITLE
Configurable log level?

### ISSUE THREAD

**Issue by theflow** [appsignal/appsignal-ruby#714]
Hi there 👋,

I'm a little bit annoyed by the log output when running on Heroku where it logs to STDOUT and then it always shows up when starting up a console:

```
$ heroku run rails c
Running rails c ... up, run.9947 (Free)
[2021-04-01T10:31:38 (process) #4][INFO] appsignal: Loading Rails (6.1.3.1) integration
[2021-04-01T10:31:40 (process) #4][INFO] appsignal: Starting AppSignal 3.0.2 (/app/bin/rails, Ruby 2.6.6, x86_64-linux)
[2021-04-01T10:31:40 (process) #4][INFO] appsignal: Installing action_mailer hook
[2021-04-01T10:31:40 (process) #4][INFO] appsignal: Installing active_job hook
[2021-04-01T10:31:40 (process) #4][INFO] appsignal: Installing active_support_notifications hook
[2021-04-01T10:31:40 (process) #4][INFO] appsignal: Installing net_http hook
[2021-04-01T10:31:40 (process) #4][INFO] appsignal: Installing puma hook
[2021-04-01T10:31:40 (process) #4][INFO] appsignal: Installing rake hook
[2021-04-01T10:31:40 (process) #4][INFO] appsignal: Installing redis hook
[2021-04-01T10:31:40 (process) #4][INFO] appsignal: Installing sidekiq hook
Loading production environment (Rails 6.1.3.1)
irb(main):001:0> 
```

Would be really great if I could set the log level to `ERROR` as part of the config, either as a separate flag like `config[:debug]` or pass in the level directly.

Cheers,
Florian

**Comment by tombruijn** [comment]
Duplicate of https://github.com/appsignal/appsignal-ruby/issues/349

**Comment by tombruijn** [comment]
This has been fixed in Ruby gem 3.0.4.

**Comment by theflow** [comment]
thank you 🙏
